flink-streaming-java/src/main/java/org/apache/flink/streaming/api/graph/StreamGraph.java

```diff
@@ -1008,12 +1008,12 @@ private void removeVertex(StreamNode toRemove) {
 
     /** Gets the assembled {@link JobGraph} with a random {@link JobID}. */
     public JobGraph getJobGraph() {
```

Comment:
Let's annotate this method with `@VisibleForTesting` because it is used only for tests.
This somehow means it's safe to use the thread classloader.